kernel/src/main/scala/org/apache/toree/boot/layer/ComponentInitialization.scala

```diff
@@ -72,25 +73,23 @@ trait StandardComponentInitialization extends ComponentInitialization {
     val (commStorage, commRegistrar, commManager) =
       initializeCommObjects(actorLoader)
 
-    val manager =  InterpreterManager(config)
-    val scalaInterpreter = manager.interpreters.get("Scala").orNull
+    val interpreterManager =  InterpreterManager(config)
 
     val dependencyDownloader = initializeDependencyDownloader(config)
-    val pluginManager = createPluginManager(
-      config, scalaInterpreter, dependencyDownloader)
+    val pluginManager = createPluginManager(config, interpreterManager, dependencyDownloader)
 
-    val kernel = initializeKernel(
-      config, actorLoader, manager, commManager, pluginManager
-    )
+    val kernel = initializeKernel(config, actorLoader, interpreterManager, commManager, pluginManager)
 
     initializePlugins(config, pluginManager)
 
-    val responseMap = initializeResponseMap()
-
     initializeSparkContext(config, kernel, appName)
 
+    interpreterManager.initializeInterpreters(kernel)
+
+    val responseMap = initializeResponseMap()
+
     (commStorage, commRegistrar, commManager,
-      manager.defaultInterpreter.orNull, kernel,
+      interpreterManager.defaultInterpreter.orNull, kernel,
```

Comment:
Is there a reason you did `orNull` here instead of just letting it fail if there is no default interpreter? We don't want the kernel to continue if there isn't a default interpreter, right?